src/struct_info.json

```diff
@@ -1569,7 +1569,11 @@
             "__WASI_FILETYPE_CHARACTER_DEVICE",
             "__WASI_FILETYPE_DIRECTORY",
             "__WASI_FILETYPE_REGULAR_FILE",
-            "__WASI_FILETYPE_SYMBOLIC_LINK"
+            "__WASI_FILETYPE_SYMBOLIC_LINK",
+            "__WASI_CLOCKID_REALTIME",
+            "__WASI_CLOCKID_MONOTONIC",
+            "__WASI_CLOCKID_PROCESS_CPUTIME_ID",
+            "__WASI_CLOCKID_THREAD_CPUTIME_ID"
```

Comment:
Is this manually maintained?  How did you know to make this change?

Reply:
Yes, basically one adds stuff to here that one needs in JS. After adding it to here, it can be used in `cDefine('__WASI_CLOCKID_REALTIME')` calls in js libraries. If you forget to add it to this JSON it will show an error suggesting you add it.